src/__tests__/__snapshots__/TextArea.test.tsx.snap

```diff
@@ -0,0 +1,67 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Why do we need snapshots?

Reply:
We don't. I can remove this. Are we all happy to remove snapshots going forward? I asked this previously and majority asked to keep it for visual regression purposes.

Reply:
I think now that we're using Chromatic, they're not necessary.

Reply:
Chromatic is disabled atm

Reply:
Ah that's right. Ok, then let's keep them until we re-enable Chromatic.